.travis.yml

```diff
@@ -2,6 +2,14 @@ language: java
 before_install:
   - bash dev/travis-before_install.sh
 
+jdk:
+  - openjdk8
+  - openjdk11
+
+matrix:
+  allow_failures:
```

Comment:
In my opinion, it does not make sense to allow the failures for the tool we are using to catch failures. I would suggest removing this part and hold this PR until the related issues are fixed.

Reply:
Makes sense @gszadovszky

Reply:
I can rebase often to check when the JDK11 becomes green.